client/src/js/components/bhDepotSelectSearch/bhDepotSelectSearch.html

```diff
@@ -0,0 +1,24 @@
+<div ng-form="$ctrl.formName" bh-multiple-depot-seach-select>
```

Comment:
Looks like a typo here.  I think this should be `bh-multiple-depot-search-select` not `seach`